src/bindings/python/fluxacct/accounting/job_archive_interface.py

```diff
@@ -222,6 +222,22 @@ def append_to_where(where_stmt, conditional):
     return job_records
 
 
+def update_t_inactive(acct_conn, last_t_inactive, user, bank):
```

Comment:
This is a really good catch and cleanup.  Can you also add a test case for this to ensure it doesn't regress?

Reply:
Yeah, absolutely. I went ahead and added a test (`test_13_update_t_inactive_success`) that checks the last seen timestamp for a user before and after their job usage factor gets calculated to ensure that it gets updated.